src/libcephfs.cc

```diff
@@ -34,11 +34,15 @@
 
 #include "include/cephfs/libcephfs.h"
 
+#define DEFAULT_UMASK 002
+
+static mode_t umask_cb(void *);
 
 struct ceph_mount_info
 {
+  mode_t umask = DEFAULT_UMASK;
 public:
-  explicit ceph_mount_info(CephContext *cct_)
+   explicit ceph_mount_info(CephContext *cct_)
```

Comment:
why change?

Reply:
That was by mistake. Will revert that change.